dev/VSIX/ProjectTemplates/Desktop/CppWinRT/SingleProjectPackagedApp/App.cpp

```diff
@@ -1,48 +1,42 @@
-// Copyright (c) Microsoft Corporation and Contributors.
-// Licensed under the MIT License.
-
 #include "pch.h"
-
 #include "App.xaml.h"
 #include "MainWindow.xaml.h"
 
 using namespace winrt;
-using namespace Windows::Foundation;
 using namespace Microsoft::UI::Xaml;
-using namespace Microsoft::UI::Xaml::Controls;
-using namespace Microsoft::UI::Xaml::Navigation;
-using namespace $safeprojectname$;
-using namespace $safeprojectname$::implementation;
 
 // To learn more about WinUI, the WinUI project structure,
 // and more about our project templates, see: http://aka.ms/winui-project-info.
 
-/// <summary>
-/// Initializes the singleton application object.  This is the first line of authored code
-/// executed, and as such is the logical equivalent of main() or WinMain().
-/// </summary>
-App::App()
+namespace $safeprojectname$::implementation
 {
-    InitializeComponent();
+    /// <summary>
+    /// Initializes the singleton application object.  This is the first line of authored code
+    /// executed, and as such is the logical equivalent of main() or WinMain().
+    /// </summary>
+    App::App()
+    {
+        InitializeComponent();
```

Comment:
Remove this one too

Reply:
I'm not sure what you're currently doing works. You should probably make an InitializeComponent function which calls the base one and then sets up the UnhandledException handler. Also, avoid passing `this` in the lambda?

Reply:
A custom InitializeComponent member function is only required to access XAML elements part of markup. Events can be set up in the constructor. Not sure why `this` was captured by the lambda before.